### ISSUE TITLE
Help with features not working

### ISSUE THREAD

**Issue by khaledh** [JakeBecker/vscode-elixir-ls#34]
I installed the extension, and it seems to detect elixir files and enables syntax highlighting, but that's about it. Features like "Go to definition" results in "No definition found for xyz"; code completion is not working; documentation lookup on hover not working. Code formatting seems to be working though, and I haven't tried debugging yet (trying to get the basics working first).

Any ideas what I might be missing?

```
> elixir --version
Erlang/OTP 20 [erts-9.0] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10]

Elixir 1.6.0 (compiled with OTP 19)
```

Platform: Windows 10
VS Code: 1.19.2
ElixirLS: 0.2.9

**Comment by khaledh** [comment]
I narrowed it down a bit, the following features are working:

- Syntax highlighting
- Code formatter (was able to format document and see reformatted code)
- Debugger (was able to hit a breakpoint inside a test and inspect variables)
- Documentation lookup on hover ONLY for Elixir kernel keywords e.g. `defmodule`, `alias`, etc.

The following features are not working:

- Go-to-definition
- Code completion
- Documentation lookup on hover for anything other than Elixir kernel
- Dialyzer analysis
- Reporting of build warnings and errors

**Comment by JakeBecker** [comment]
Apologies -- builds are broken on Windows, and these features depend on
working builds. I have a fix ready but haven't done a release yet. Will
release tomorrow probably. I meant to do it last weekend but totally
forgot. My bad!

**Comment by khaledh** [comment]
No worries. Thanks for the quick response! Looking forward to the fixed release.

**Comment by CristianOspina** [comment]
Any news on this? 😅

**Comment by JakeBecker** [comment]
Just published v0.2.10 with the fix. Sorry for the wait -- there were other bugs in Windows having to do with the conversion between paths and file URIs. Should be working now!
